### ISSUE TITLE
Crafting with a single Fresh Water gives 8 Fresh Waters

### ISSUE THREAD

**Issue by merdink** [sibomots/FooFarm#14]
If you put a single Fresh Water item in the crafting grid, it will give you 8 of them.
![image](https://user-images.githubusercontent.com/4110814/106544797-26b35d00-64d6-11eb-9553-5fa4f8b010bb.png)

**Comment by sibomots** [comment]
Bug or feature?  :-)   

So what's the expected behavior?

**Comment by merdink** [comment]
I don't know. Milk doesn't do it. If it doesn't impact the challenge of the mod pack then I guess we can keep it as is.

**Comment by Drakonas** [comment]
I mean, water is technically infinitely accessible anyway. But yeah, I don't think that's intended behavior to make infinite fresh waters. But I think this would be a mod-level bug, not server-level.

**Comment by merdink** [comment]
I feel silly. After looking at the source code for the mod, I saw a reference to it in the config file. I changed the setting and it seems that the whole thing is intended. So what we need to decide is, should we keep it on or off?

harvestcraft.cfg
Under "miscellaneous recipes"
Entry is `B:enablelistAllwaterfreshwater=true`
True means Fresh Water can be used to make more,

How do we figure out if its bad for the difficulty of the pack?
